### PR TITLE
chore: stop removing `fuel-ts/utils` node modules on unit test

### PR DESCRIPTION
**Pull request by Torres-ssf**
Fixes the following error that is shown after running the tests 2 times in a row:

Screenshot 2024-01-29 at 11 11 48

### CODE REVIEW

**Review by danielbate — APPROVED** [review]
🦊

**Review by arboleya — COMMENTED** [review]
Still [failing](https://github.com/FuelLabs/fuels-ts/actions/runs/7701818945/job/20988743046?pr=1692). 🤔

File: packages/utils/src/cli-utils/findBinPath.test.ts [hunk 1]
```diff
@@ -40,15 +40,6 @@ describe('findBinPath', () => {
...
 
-  it('should find bin path two dir up', () => {
-    const base = join(__dirname, '..', '..'); // two dirs up
-    const { cmdName, cmdPath, resetDisk } = bootstrap(base);
-    const binPath = findBinPath(cmdName, base);
-
-    resetDisk();
-    expect(binPath).toEqual(cmdPath);
-  });
-
```
Comment (arboleya) [inline packages/utils/src/cli-utils/findBinPath.test.ts:51]: Did you remove this test by mistake?

Comment (Torres-ssf) [reply packages/utils/src/cli-utils/findBinPath.test.ts:51]: @arboleya this unit test is the source of the problem. 

It is deleting the node_modules from the utils package. 

That's why running the tests 2 times in a row result in an error.

**Comment by Torres-ssf** [comment]
> Still [failing](https://github.com/FuelLabs/fuels-ts/actions/runs/7701818945/job/20988743046?pr=1692). 🤔

@arboleya This PR addresses another problem. 

Right now running `pnpm ci:test` 2 times in a row does not work because there is a unit test removing the `node_modules` from the `utils` package

**Review by arboleya — APPROVED** [review]

**Comment by arboleya** [comment]
@Torres-ssf When I approved this PR, the "Rerun FE Conditional" [broke](https://github.com/FuelLabs/fuels-ts/actions/runs/7731857968/job/21080474624?pr=1692) again. 🤔

**Comment by Torres-ssf** [comment]
> @Torres-ssf When I approved this PR, the "Rerun FE Conditional" [broke](https://github.com/FuelLabs/fuels-ts/actions/runs/7731857968/job/21080474624?pr=1692) again. 🤔

Yes, I've seen this happening one time. I reckon this is happening because it tries to rerun `FE Conditonal Review` when it is already running.

But I am not sure why `FE Conditional Review` was triggered in the first place

**Review by Dhaiwat10 — APPROVED** [review]

**Pull request merged**